docs/install/Knative-with-any-k8s.md

```diff
@@ -1,89 +1,470 @@
 ---
-title: "Install on a Kubernetes cluster"
-linkTitle: "On existing cluster"
+title: "Installing Knative"
+linkTitle: "Installing Knative"
 weight: 15
 type: "docs"
 ---
 
-This guide walks you through the installation of the latest version of Knative
-using pre-built images.
+This guide walks you through the installation of the latest version of Knative.
+
+Knative has several pieces, which can be installed and used independently or together.
+To help you pick and choose the pieces that are right for you, here is a brief
+description of each:
+ - [**Serving**](#installing-the-serving-component)(_stable_) provides an abstraction for stateless request-based scale-to-zero services.
+ - [**Eventing**](#installing-the-eventing-component)(_alpha_) provides abstractions to enable binding event sources (e.g. Github Webhooks, Kafka) and consumers (e.g. Kubernetes or Knative Services).
+ - [**Monitoring**](#installing-the-monitoring-bundle)(_alpha_) provides a bundle of standard tooling that can be used to get visibility into the health of the software running on Knative.
 
 ## Before you begin
 
+This guide assumes that you want to install an upstream Knative release on a Kubernetes cluster. A growing number of vendors are offering managed Knative services, see [here](./vendor-options.md) for a list of the known offerings.
+
 Knative requires a Kubernetes cluster v1.15 or newer, as well as a compatible
```

Comment:
```suggestion
Knative < version > requires a Kubernetes cluster v1.15 or newer, as well as a compatible
```

It would also be nice to have a consolidated < min_k8s_version > variable.